R/pkg/R/DataFrame.R

```diff
@@ -677,13 +677,17 @@ setMethod("unique",
 #' collect(sample(df, TRUE, 0.5))
 #'}
 setMethod("sample",
-          # TODO : Figure out how to send integer as java.lang.Long to JVM so
-          # we can send seed as an argument through callJMethod
           signature(x = "DataFrame", withReplacement = "logical",
                     fraction = "numeric"),
-          function(x, withReplacement, fraction) {
+          function(x, withReplacement, fraction, seed) {
```

Comment:
@felixcheung Shouldn't we document this param in the roxygen doc above ? Otherwise how would anybody know we support a `seed` ?